libs/designsystem/src/lib/components/chip/chip.component.scss

```diff
@@ -22,8 +22,10 @@
   margin: size('xxxs');
   cursor: pointer;
   user-select: none;
+  position: relative;
   @include theme-background('white');
   @include selected-and-hover('black');
+  @include increase-target-size();
```

Comment:
Mixin includes with no content block should be declared before local declarations